examples/app-ofa-private/README.md

```diff
@@ -10,7 +9,8 @@ User transactions are stored in the confidential datastore and only a small hint
 Run `Suave` in development mode:
 
 ```
-$ suave --suave.dev
+$ cd ../..
+$ docker-compose up
```

Comment:
switch back to docker compose? which directory is `../..` supposed to be?

Reply:
Yes, we use docker compose to enable the Suave<>L1 integration. The `../..` directory is supposed to mean the root directory of suapp-examples.